templates/user/dashboard/dashboard.tmpl

```diff
@@ -6,7 +6,15 @@
 		<div class="ui mobile reversed stackable grid">
 			<div class="ui container ten wide column">
 				{{template "user/heatmap" .}}
-				{{template "user/dashboard/feeds" .}}
+				{{if eq (len .Feeds) 0}}
+					<div class="gt-df gt-fc gt-ac gt-px-5 activity-empty">
```

Comment:
Can this use the `empty-placeholder` class?

Reply:
Move `.empty-placeholder`to `base.css`. I think we can use `.empty-placeholder` here, and then add its own padding.